### PR TITLE
Eco 1149/support dark mode

### PR DESCRIPTION
**Pull request by OrenZak**
#### Main purpose:
Allow the developer to choose between light/ dark theme
#### Technical description:
- Added `KinTheme`
- Create two styles with different attrs values according to the theme name. The attrs are the difference between the light and dark modes.

### CODE REVIEW

File: core/src/main/java/com/kin/ecosystem/core/data/internal/ConfigurationLocalImpl.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    private val configurationSharedPref: SharedPreferences
+
+    override var kinTheme: KinTheme?
+        get() = KinTheme.valueOf(configurationSharedPref.getString(KIN_THEME_KEY, KinTheme.LIGHT.name))
+        set(value) {
+            with(configurationSharedPref.edit()) {
+                kinTheme?.let { currentTheme ->
+                    if(currentTheme != value ) {
+                        value?.let { value ->
+                            putString(KIN_THEME_KEY, value.name).apply()
```
Comment (shayb) [inline core/src/main/java/com/kin/ecosystem/core/data/internal/ConfigurationLocalImpl.kt:18]: no defualt value?

**Review by shayb — APPROVED** [review]
look at my comment

File: sdk/src/main/res/drawable/kinecosystem_ic_arrow_right.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
```
Comment (shayb) [inline sdk/src/main/res/drawable/kinecosystem_ic_arrow_right.xml:6]: are you sue vectors supported ?? did you check on device with minimum sdk

Comment (OrenZak) [reply sdk/src/main/res/drawable/kinecosystem_ic_arrow_right.xml:6]: yup works check on API 19:
ref:  https://developer.android.com/guide/topics/graphics/vector-drawable-resources

File: core/src/main/java/com/kin/ecosystem/core/data/internal/ConfigurationLocalImpl.kt (hunk 1 above)
Comment (OrenZak) [reply core/src/main/java/com/kin/ecosystem/core/data/internal/ConfigurationLocalImpl.kt:18]: there is, check the `get():` property

**Pull request merged**